src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj

```diff
@@ -57,8 +57,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.Common.Authentication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.3.3-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
-      <Private>True</Private>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.4.1-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
```

Comment:
I will do that once my PR for AutoRest is published. The problem is that there are more changes that need to happen in PowerShell and autorest for that to work, since my package is based off version 1.3.3 and version 1.3.4 breaks the powershell. As a result, all my packages will be private until we are at a point where everything is properly published and ready to be upgraded in PowerShell.